docs/server/commands.rst

```diff
@@ -419,6 +419,49 @@ Available options:
   (quotes included).
 
 
+.. django-admin:: set_filetype
+
+set_filetype
+^^^^^^^^^^^^
+
+.. versionadded:: 2.8
+
+This command sets file formats for projects, and also allows to convert files
+to another format.
+
+.. django-admin-option:: --from-filetype
+
+Specifies to convert stores of this file type.
+
+.. django-admin-option:: --matching
+
+Specifies which stores to convert by matching their path within the project,
+excluding file extension.
+
+`--from-filetype` and `--matching` can be both specified at the same time.
+
+
+For example, to add the `properties` format to a project, run:
+
+.. code-block:: console
+
+    $ pootle set_filetype --project=myproj properties
+
+
+To convert stores of `po` format to `properties`, run:
+
+.. code-block:: console
+
+    $ pootle set_filetype --project=myproj --from-filetype=po properties
+
+
+For example, to add a new format to a project, run:
+
+.. code-block:: console
+
+    $ pootle set_filetype --project=myproj --matching=mydir/myfile-* properties
```

Comment:
I'm not 100% sure I understand `--matching` based on this.  I'd assume I'd use `--matching` to convert certain files to a new filetype.  I.e. why didn't I just use your first example to achieve what you've specified here?